react-design/my-app/src/RotatingBanner.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import { AccordionButtons } from './AccordionButtons';
+import { Banner } from './Banner';
+import { NextButton } from './NextButton';
+import { PrevButton } from './PrevButton';
+
+type Props = {
+  items: string[];
+};
+
+export function RotatingBanner({ items }: Props) {
+  const [index, setIndex] = useState(0);
+
+  function handleNext() {
+    if (index === 5) {
```

Comment:
It would be best to compare to `items.length - 1` so that the code works if the number of items changes.